backend/apps/github/common.py

```diff
@@ -227,3 +232,61 @@ def sync_repository(
     )
 
     return organization, repository
+
+
+def sync_issue_comments(gh_client: Github, issue: Issue):
+    """Sync new comments for a mentorship program specific issue on-demand.
+
+    Args:
+        gh_client (Github): GitHub client.
+        issue (Issue): The local database Issue object to sync comments for.
+
+    """
+    logger.info("Starting comment sync for issue #%s", issue.number)
+
+    try:
+        if not (repository := issue.repository):
+            logger.warning("Issue #%s has no repository, skipping", issue.number)
+            return
+
+        logger.info("Fetching repository: %s", repository.path)
+
+        gh_repository = gh_client.get_repo(repository.path)
+        gh_issue = gh_repository.get_issue(number=issue.number)
+
+        since = issue.comments.order_by("-updated_at").values_list(
```

Comment:
You keep adding ad-hoc variables instead of updating `issue::latest_comment` code. Why?